ci/patch_src_in_place

```diff
@@ -35,4 +35,7 @@ then
     done
 fi
 
+# Format the go code correctly.
+find src/control/ -name '*.go' -and -not -path '*vendor*'| xargs gofmt -w
```

Comment:
(lint) Use -print0/-0 or -exec + to allow for non-alphanumeric filenames. [SC2038]